systemtest/src/test/java/io/strimzi/systemtest/cruisecontrol/CruiseControlIsolatedST.java

```diff
@@ -159,9 +159,11 @@ void testCruiseControlTopicExclusion(ExtensionContext extensionContext) {
 
         LOGGER.info("Checking status of KafkaRebalance");
         KafkaRebalanceStatus kafkaRebalanceStatus = KafkaRebalanceResource.kafkaRebalanceClient().inNamespace(NAMESPACE).withName(clusterName).get().getStatus();
-        assertThat(kafkaRebalanceStatus.getOptimizationResult().get("excludedTopics").toString(), containsString(excludedTopic1));
-        assertThat(kafkaRebalanceStatus.getOptimizationResult().get("excludedTopics").toString(), containsString(excludedTopic2));
-        assertThat(kafkaRebalanceStatus.getOptimizationResult().get("excludedTopics").toString(), not(containsString(includedTopic)));
+        Map<String, Object> optimizationResult = kafkaRebalanceStatus.getOptimizationResult();
+        Map<String, Object> optimizationSummary = (Map<String, Object>) optimizationResult.get("summary");
```

Comment:
As commented above, currently the optimization result is not under an additional "summary" key and your code puts it under one, which it shouldn't. After you move it back to be the map itself you will need to change this code as well.